core/include/traccc/edm/seed.hpp

```diff
@@ -24,6 +25,25 @@ struct seed {
     scalar weight;
     scalar z_vertex;
 
+    TRACCC_HOST_DEVICE
+    std::array<measurement_link, 3> get_measurements(
+        const spacepoint_collection_types::const_view& spacepoints_view,
+        const cell_module_collection_types::host& modules) const {
+        const spacepoint_collection_types::const_device spacepoints(
+            spacepoints_view);
+        alt_measurement alt_measB = spacepoints.at(spB_link).meas;
```

Comment:
Why does this have to be done via the ```alt_measurement``` type?

Reply:
BTW, I think we will remove current `measurement` and rename `alt_measurement` to `measurement`

Reply:
that's the idea :) if you need any help with that you can @ me, but it should be pretty trivial